src/Calculator/Views/DateCalculator.xaml

```diff
@@ -502,7 +502,7 @@
                                 Grid.Row="2"
                                 Margin="0,0,0,0"
                                 Style="{StaticResource DateCalculation_CalendarPickerStyle}"
-                                AutomationProperties.LabeledBy="{x:Bind AddSubtract_Date_FromLabel}"
+                                AutomationProperties.LabeledBy="{x:Bind Date_FromLabel}"
```

Comment:
`CalendarDatePicker` supports [setting header text](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.CalendarDatePicker#setting-a-header-and-placeholder-text), which would collapse this `TextBox` + `CalendarDatePicker` into a single element (and address the accessibility bug as well). I would consider this approach instead of using a custom header `TextBlock` with `LabeledBy`. If this works out, we could simplify the two other instances of `CalendarDatePicker` on this page by leveraging this approach as well.